src/desdemona/ui/table.cljs

```diff
@@ -1,5 +1,39 @@
-(ns desdemona.ui.table)
+(ns desdemona.ui.table
+  (:require [wilson.dom :as wd]
+            [reagent.session :as session]))
+
+(def rows (:results @session/state))
```

Comment:
Here's your bug. This will get evaluated at compile time, when session/state is still empty. It's only when figwheel does its magic that it will get evaluated afterwards! put this expression inside table-component.